.github/workflows/build-test-release.yml

```diff
@@ -19,22 +19,43 @@ jobs:
         uses: actions/checkout@v2
         with:
           fetch-depth: 0
+          path: subnet-cli
       - name: Set up Go
         uses: actions/setup-go@v2
         with:
-          go-version: 1.17
+          go-version: 1.18
       - name: Run unit tests
         shell: bash
-        run: scripts/tests.unit.sh
+        run: |
+          cd subnet-cli
+          scripts/tests.unit.sh
       - name: Run e2e tests
         shell: bash
-        run: scripts/tests.e2e.sh 1.7.6
+        run: |
```

Comment:
Q. If we 1.9 both for e2e and go module, would it break?

Reply:
did not tried, just used the avago bumped by anr bump. will check. e2e should work.

Reply:
worked. bumping